src/main/java/org/quiltmc/asmr/processor/AsmrProcessorRunner.java

```diff
@@ -1,30 +1,15 @@
 package org.quiltmc.asmr.processor;
 
 import org.jetbrains.annotations.Nullable;
-import org.quiltmc.asmr.processor.capture.AsmrCapture;
-import org.quiltmc.asmr.processor.capture.AsmrNodeCapture;
-import org.quiltmc.asmr.processor.capture.AsmrReferenceCapture;
-import org.quiltmc.asmr.processor.capture.AsmrReferenceSliceCapture;
-import org.quiltmc.asmr.processor.capture.AsmrSliceCapture;
+import org.quiltmc.asmr.processor.capture.*;
```

Comment:
Please don't replace with wildcard imports

Reply:
Whoops, IntelliJ I enabled auto import optimizations but forgot to disable the star imports